torch/export/_unlift.py

```diff
@@ -273,6 +274,7 @@ def _create_stateful_graph_module(
 
 
 def _unlift_exported_program_lifted_states(ep: ExportedProgram) -> torch.nn.Module:
+    ep = _remove_effect_tokens(ep)
```

Comment:
What is "unlifting" and where does it happen?

Reply:
Unlifting basically takes the exported program and returns a torch.fx.GraphModule with the parameters/buffers unlifted from inputs and as get_attr nodes, and any mutations put back into the graph with copy_. Users can call that eagerly and do what they would do to GraphModules returned by symbolic_trace.

Reply:
sg!